diff -ru shell-color-scripts-source/colorscript.sh shell-color-scripts-patched/colorscript.sh
--- shell-color-scripts-source/colorscript.sh	2023-02-25 00:49:51.753335423 -0300
+++ shell-color-scripts-patched/colorscript.sh	2023-02-25 01:39:46.756760828 -0300
@@ -5,7 +5,7 @@
 if [[ "$DEV" -gt 0 ]]; then
     DIR_COLORSCRIPTS="./colorscripts"
 else
-    DIR_COLORSCRIPTS="/opt/shell-color-scripts/colorscripts"
+    DIR_COLORSCRIPTS="/usr/libexec/shell-color-scripts/colorscripts"
 fi
 
 if command -v find &>/dev/null; then
diff -ru shell-color-scripts-source/completions/_colorscript shell-color-scripts-patched/completions/_colorscript
--- shell-color-scripts-source/completions/_colorscript	2023-02-25 00:49:51.763335424 -0300
+++ shell-color-scripts-patched/completions/_colorscript	2023-02-25 01:38:07.470091096 -0300
@@ -1,6 +1,6 @@
 #compdef colorscript
 
-DIR_COLORSCRIPTS="/opt/shell-color-scripts/colorscripts"
+DIR_COLORSCRIPTS="/usr/libexec/shell-color-scripts/colorscripts"
 if command -v find &>/dev/null; then
     LS_CMD="$(command -v find) ${DIR_COLORSCRIPTS} -maxdepth 1 -type f | xargs -I$ basename $"
 else
diff -ru shell-color-scripts-source/completions/colorscript.fish shell-color-scripts-patched/completions/colorscript.fish
--- shell-color-scripts-source/completions/colorscript.fish	2023-02-25 00:49:51.763335424 -0300
+++ shell-color-scripts-patched/completions/colorscript.fish	2023-02-25 01:39:06.000092901 -0300
@@ -3,7 +3,7 @@
 # according to https://fishshell.com/docs/current/completions.html
 # function
 
-set dir_colorscripts "/opt/shell-color-scripts/colorscripts"
+set dir_colorscripts "/usr/libexec/shell-color-scripts/colorscripts"
 set dir_blacklisted_colorscripts "$dir_colorscripts/blacklisted"
 
 # Lists all filenames in a given directory
@@ -75,4 +75,4 @@
 
 # Blacklisted colorscript name autocompletions
 set -l commands_that_take_blacklisted_names -u --unblacklist unblacklist
-complete -c colorscript -n "__fish_seen_subcommand_from $commands_that_take_blacklisted_names" -a '(blacklisted_cs_names)'
\ No hay ningún carácter de nueva línea al final del fichero
+complete -c colorscript -n "__fish_seen_subcommand_from $commands_that_take_blacklisted_names" -a '(blacklisted_cs_names)'
diff -ru shell-color-scripts-source/Makefile shell-color-scripts-patched/Makefile
--- shell-color-scripts-source/Makefile	2023-02-25 00:49:51.753335423 -0300
+++ shell-color-scripts-patched/Makefile	2023-02-25 01:15:56.113383475 -0300
@@ -1,12 +1,11 @@
-clean: 
-	rm -rf /opt/shell-color-scripts
-
-install: clean
-	mkdir -p /opt/shell-color-scripts/colorscripts
-	cp -rf colorscripts/* /opt/shell-color-scripts/colorscripts
-	cp colorscript.sh /usr/local/bin/colorscript
+install:
+	install -Dm755 completions/_colorscript ${D}/usr/share/zsh/_colorscript
+	install -Dm755 completions/colorscript.fish ${D}/usr/share/fish/vendor_completions.d/colorscript.fish
+	install -Dm755 colorscripts/* -t ${D}/usr/libexec/shell-color-scripts/colorscripts
+	install -Dm755 colorscript.sh ${D}/usr/bin/colorscript
 
 uninstall:
-	rm -rf /opt/shell-color-scripts
-	rm -f /usr/local/bin/colorscript
-
+	rm -f /usr/bin/colorscript
+	rm -rf /usr/libexec/shell-color-scripts
+	rm -f /usr/share/zsh/_colorscript
+	rm -f /usr/share/fish/vendor_completions.d/colorscript.fish
